docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -1367,6 +1370,60 @@ and annotating it with `@PersistenceUnitExtension` (or `@PersistenceUnitExtensio
 will replace the current Quarkus default implementation `io.quarkus.hibernate.orm.runtime.tenant.DataSourceTenantConnectionResolver`.
 Your custom connection resolver would allow for example to read tenant information from a database and create a connection per tenant at runtime based on it.
 
+[CAUTION]
+====
+<<automatic-integration,Automatic integrations>> will **not** work with programmatically-created `ConnectionProvider`.
```

Comment:
`automatic-integration` doesn't resolve. What did you want to link to exactly?

Reply:
This anchor in the same page: https://quarkus.io/guides/hibernate-orm#automatic-integration

Reply:
Thanks.